packages/common/src/ui/layout/tables/columns/PaginationRow/PaginationRow.test.tsx

```diff
@@ -149,36 +149,6 @@ describe('PaginationRow', () => {
     expect(onChange).toBeCalledWith(0);
   });
 
-  it('does not trigger the callback when the ellipsis is pressed', () => {
-    const offset = 0;
-    const first = 10;
-
-    const total = 20000;
-    const onChange = jest.fn();
-
-    const { getByLabelText } = render(
-      <TestingProvider>
-        <TableProvider createStore={createTableStore}>
-          <PaginationRow
-            page={0}
-            offset={offset}
-            total={total}
-            first={first}
-            onChange={onChange}
-          />
-        </TableProvider>
-      </TestingProvider>
-    );
-
-    const node = getByLabelText(/ellipsis/i);
-
-    act(() => {
-      userEvent.click(node);
-    });
-
-    expect(onChange).toBeCalledTimes(0);
-  });
-
```

Comment:
Uhh. This test was to prevent the ellipsis of the pagination row being clickable. Somehow it's started to fail and now I can't 'find' the ellipsis button with the dom elements and didn't want to spend more time making it work.....

Reply:
odd. it used to be a button, now just a `div` 🤷